errors after atom-autogrow-textarea 6.0.1 update

```shell
/src/Textarea/index.css:1:1: Failed to find '@cyclic/atom-autogrow-textarea/src/atom-autogrow-textarea'
```
```shell
Error: Cannot find module '@cyclic/atom-autogrow-textarea/src/atom-autogrow-textarea' from '/src/Textarea'
```

--- a/demo/index.js
+++ b/demo/index.js
@@ -56,12 +56,12 @@ function demo({DOM}) {
     Textarea({
       DOM, props$: Rx.Observable.just({
         label: `textarea with initial value`,
-        value: `Hi,\r\n
-        \r\n
-        I am filled with some random text.\r\n
-        \r\n
-        Regards,\r\n
-        The author`,
+        value: `Hi,
+
+I am filled with some forgettable text.
+
+Regards,
+The Author`,
       }),
     }).DOM,
     Textarea({

--- a/package.json
+++ b/package.json
@@ -29,14 +29,14 @@
     ]
   },
   "dependencies": {
-    "@cyclic/atom-autogrow-textarea": "5.0.0",
+    "@cyclic/atom-autogrow-textarea": "6.0.1",
     "@cyclic/atom-color": "1.0.0",
     "@cyclic/atom-flex-layout": "1.0.0",
     "@cyclic/atom-layout": "1.0.2",
     "@cyclic/atom-styles": "1.0.0",
     "@cyclic/atom-typography": "1.0.0",
-    "@cyclic/util-combine-class-names": "2.0.0",
-    "categories-js": "^0.3.0",
+    "@cyclic/util-combine-class-names": "3.0.0",
+    "categories-js": "0.3.0",
     "cuid": "1.3.8",
     "ramda": "^0.17.1"
   },

--- a/src/Textarea/index.css
+++ b/src/Textarea/index.css
@@ -1,4 +1,4 @@
-@import "@cyclic/atom-autogrow-textarea/src/atom-autogrow-textarea";
+@import "@cyclic/atom-autogrow-textarea/src/AutogrowTextarea";
 
 .molecule-Textarea {
   display: block;

--- a/src/Textarea/index.js
+++ b/src/Textarea/index.js
@@ -1,5 +1,5 @@
-import atomAutogrowTextarea
-  from '@cyclic/atom-autogrow-textarea/src/atom-autogrow-textarea';
+import AutogrowTextarea
+  from '@cyclic/atom-autogrow-textarea/src/AutogrowTextarea';
 import cuid from 'cuid';
 import makeInputContainer from './../shared/makeInputContainer';
 import model from './../shared/model';
@@ -16,7 +16,7 @@ function Textarea(sources) {
   const id = cuid();
   const state$ = model({componentName: COMPONENT_NAME, props$});
 
-  const textareaDOM = atomAutogrowTextarea({
+  const textareaDOM = AutogrowTextarea({
     DOM, props$: state$.map((state) => clone(state)),
   }).DOM;
 

